XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/InputSystem/Simulation/IMixedRealityInputSimulationDataProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Interfaces.Providers.InputSystem.Simulation
+{
+    public interface IMixedRealityInputSimulationDataProvider : IMixedRealityDataProvider
+    {
+
```

Comment:
Placeholder for now?

Reply:
I need the interface so the Inspector lists fitting data providers in the inspector when adding them to the data providers profile